tools/rime_table_decompiler.cc

```diff
@@ -0,0 +1,105 @@
+// rime_table_decompiler.cc
+// nopdan <me@nopdan.com>
+//
+#include <cmath>
+#include <fstream>
+#include <iomanip>
+#include <ios>
+#include <iostream>
+#include <rime/dict/table.h>
+
+using namespace std;
```

Comment:
For a serious program this is not acceptable. Please use full `std::ofstream` etc.